tethys_apps/base/app_base.py

```diff
@@ -68,131 +72,235 @@ def url_maps(self):
 
             from tethys_sdk.base import url_map_maker
 
-            def url_maps(self):
-                \"""
-                Example url_maps method.
-                \"""
-                # Create UrlMap class that is bound to the root url.
-                UrlMap = url_map_maker(self.root_url)
+            class MyFirstApp(TethysAppBase):
 
-                url_maps = (UrlMap(name='home',
-                                   url='my-first-app',
-                                   controller='my_first_app.controllers.home'
-                                   ),
-                )
+                def url_maps(self):
+                    \"""
+                    Example url_maps method.
+                    \"""
+                    # Create UrlMap class that is bound to the root url.
+                    UrlMap = url_map_maker(self.root_url)
 
-                return url_maps
+                    url_maps = (UrlMap(name='home',
+                                       url='my-first-app',
+                                       controller='my_first_app.controllers.home',
+                                       ),
+                    )
+
+                    return url_maps
         """
         raise NotImplementedError()
 
-    def persistent_stores(self):
+    # TODO: ADD SETTING LINK TO TOP OF APP WHEN LOGGED IN AS STAFF
+    def custom_settings(self):
```

Comment:
I think it would be beneficial to add commented out stubs for these new settings methods to the app.py_tmpl to give uses an easy reference when generating a new app from the scaffold.